x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexing/IndexerJobStats.java

```diff
@@ -24,45 +23,48 @@
  * and are only for external monitoring/reference.  Statistics are not persisted with the job, so if the
  * allocated task is shutdown/restarted on a different node all the stats will reset.
  */
-public class RollupJobStats implements ToXContentObject, Writeable {
+public class IndexerJobStats implements ToXContentObject, Writeable {
 
     public static final ParseField NAME = new ParseField("job_stats");
 
     private static ParseField NUM_PAGES = new ParseField("pages_processed");
-    private static ParseField NUM_DOCUMENTS = new ParseField("documents_processed");
-    private static ParseField NUM_ROLLUPS = new ParseField("rollups_indexed");
+    private static ParseField NUM_INPUT_DOCUMENTS = new ParseField("documents_processed");
+    // BWC for RollupJobStats
+    public static String ROLLUP_BWC_XCONTENT_PARAM = "rollup_6_4_compat";
```

Comment:
I'm not in love with this flag name... but I can't think of something better.  It will only be exposed on the GetRollupJob API, so I don't think we need "rollup" in the name.

Maybe `"old_stats_format"` or something? Open to suggestions